README.md

```diff
@@ -12,33 +12,55 @@
 The project's architecture deploys a server-less backend, AWS Lambda, that compiles and executes user-submitted Coconut code. The frontend is built with React. For local development, a Flask app serves as the backend.
 
 ### Prerequisites
+__Backend and package manager__
 * [Python >= 3](https://www.python.org)
-* [Node >= 6](https://nodejs.org)
+* [Node >= 6, <=9](https://nodejs.org)
 * [Yarn >= 1](https://yarnpkg.com/en/docs/install)
 
-### Python Packages
+__Notable Python Packages__ (installed with `pip install -r requirements.txt` below)
 * [coconut](https://pypi.org/project/coconut): compiles Coconut code into Python code.
 * [flask](https://pypi.org/project/Flask): serves backend API for local development.
 * [zappa](https://pypi.org/project/zappa): packages and deploys Flask app to AWS Lambda.
 
 ## Installation
-```bash
-# Backend
-pip install -r requirements.txt
+For backend installation, we recommend you use Anaconda to handle Python packages. If Python 3.x is not your default Python version, you may have to use `pip3` instead of `pip` to install Python packages and `python3` instead of `python` to run Python. We recommend you check this by using `which python` and `which pip` commands. If users have Python from Anaconda, we further recommend users install all Python packages within a new virtual environment.
```

Comment:
- Consider choosing between using second person ("you") and third person ("the users")
- "check this" meaning check the python version? wouldn't that be `python --version`, not `which python` (and `pip --version` not `which pip`?)
- Consider hyperlinking "[virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/#creating-a-virtualenv)"?
- Just a personal question (don't need to actually explain in the README itself), why do we recommend using a virtual environment and why do we only recommend it for Anaconda?